src/Symfony/Component/Uid/UuidV4.php

```diff
@@ -25,14 +25,38 @@ class UuidV4 extends Uuid
     public function __construct(string $uuid = null)
     {
         if (null === $uuid) {
+            $this->uid = self::generate();
+        } else {
+            parent::__construct($uuid);
+        }
+    }
+
+    public function getTime(): float
```

Comment:
Do we really need this? It's only used in tests

Reply:
Removed.